### ISSUE TITLE
Vim8 support

### ISSUE THREAD

**Issue by melonaerial** [jeaye/color_coded#142]
Hello @jeaye !
I think this is issue have to be here too. I know that you got issue #89  and it is maybe hard to work on this too issues together. But I think that support of Vim 8.0 have reason. What do you think about that ?

**Comment by jeaye** [comment]
I think it's good to have this issue; thanks for opening it. I think either Vim 8 is going to be supported or NeoVim is. Supporting Vim 7, Vim 8, and NeoVim, each with different approaches to messaging, sounds like something I'd like to avoid.

**Comment by melonaerial** [comment]
Yeah, that's gonna be real crazy )) I think one of them have to gone in history )))
